pkg/skaffold/deploy/label/labels.go

```diff
@@ -139,7 +139,7 @@ func resolveNamespace(ns string) (string, error) {
 		return "", fmt.Errorf("getting kubeconfig: %w", err)
 	}
 
-	current, present := cfg.Contexts[cfg.CurrentContext]
```

Comment:
I wonder how many other references we have to .CurrentContext like this.  Honestly, `kubectx.CurrentConfig()` should be banished and the cluster+context should be passed around via the context or as a separate argument.